### PR TITLE
Tshuvashit

### PR DESCRIPTION
**Pull request by rurhtshuva**

### CODE REVIEW

File: src/components/Calculator/Calculator.jsx [hunk 1]
```diff
@@ -1,39 +1,42 @@
...
 
   return (
-      <Grid columns = {4} centered>
-        <GridColumn>
-          <Card>
-          <CardContent header='Calculator A-la Shtrahman' />
+    <Grid columns={4} centered>
+      <GridColumn>
+        <Card>
+          <CardContent header=" chtok chtok ya omo" />
```
Comment (MrShtrahman) [inline src/components/Calculator/Calculator.jsx]: Nope.

File: src/components/Calculator/Buttons/CalcButton/CalcButtonType.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { SemanticSIZES, SemanticCOLORS } from "semantic-ui-react";
+import { ReactText /*MouseEvent, SyntheticEvent*/ } from "react";
+export type CalcButtonType = {
+  value: ReactText;
+  onClick: any; // can't do it for and dont know why
```
Comment (MrShtrahman) [inline src/components/Calculator/Buttons/CalcButton/CalcButtonType.ts]: Did you try `onClick: () => void` or `onClick(event: React.MouseEvent<HTMLButtonElement>): void` ?

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import Calculator from './components/Calculator/Calculator';
+import Interface from './hoc/Interface';
+import React from 'react';
+
+const App = () => {
+
+  return (
+    <Calculator />
```
Comment (MrShtrahman) [inline src/App.jsx]: Any reason for not changing `Calculator` to `.tsx`?

File: package.json [hunk 4]
```diff
@@ -35,12 +36,12 @@
     ]
   },
   "devDependencies": {
-    "@testing-library/react-hooks": "^3.2.1",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "expect": "^25.1.0",
     "jest-environment-enzyme": "^7.1.2",
     "jest-enzyme": "^7.1.2",
+    "react-hooks-testing-library": "^0.6.0",
```
Comment (MrShtrahman) [inline package.json]: Why?

File: src/components/Calculator/Calculator.jsx (hunk 1 above)
Comment (rurhtshuva) [reply src/components/Calculator/Calculator.jsx]: but ata omo :{

File: src/components/Calculator/Buttons/CalcButton/CalcButtonType.ts (hunk 2 above)
Comment (rurhtshuva) [reply src/components/Calculator/Buttons/CalcButton/CalcButtonType.ts]: fuck UR good

File: src/App.jsx (hunk 3 above)
Comment (rurhtshuva) [reply src/App.jsx]: ill let you have fun

File: package.json (hunk 4 above)
Comment (rurhtshuva) [reply package.json]: i dont know , i did npm i and it changed.., after fact chack it is Deprecated , ill change it

**Pull request merged**